cmd/occollector/app/builder/processor_builder.go

```diff
@@ -91,6 +93,16 @@ type BatchingConfig struct {
 	RemoveAfterTicks *int `mapstructure:"remove-after-ticks,omitempty"`
 }
 
+// JaegerProtoGRPCSenderCfg holds configuration for Jaeger Proto GRPC sender
+type JaegerProtoGRPCSenderCfg struct {
```

Comment:
nit: can you move this above `BatchingConfig` so that it is closer to the other sender configs?